eclass/distutils-r1.eclass

```diff
@@ -1514,11 +1514,33 @@ distutils-r1_python_install() {
 	local merge_root=
 	if [[ ${DISTUTILS_USE_PEP517} ]]; then
 		local root=${BUILD_DIR}/install
+		local wrapped_scriptdir=${root}$(python_get_scriptdir)
+
+		# we are assuming that _distutils-r1_post_python_compile
+		# has been called and ${root} has not been altered since
+		# let's explicitly verify these assumptions
+
+		# remove files that we've created explicitly
+		rm "${root}${scriptdir}"/{"${EPYTHON}",python3,python,pyvenv.cfg} || die
+		# verify that scriptdir & wrapped_scriptdir both contain
+		# the same files
+		local ck1=$(
+			{ cd "${root}${scriptdir}" && find . -type f -print0; } |
+			sort -z | cksum
+		)
+		local ck2=$(
+			{ cd "${wrapped_scriptdir}" 2>/dev/null &&
+				find . -type f -print0; } |
+			sort -z | cksum
+		)
+		if [[ ${ck1} != ${ck2} ]]; then
```

Comment:
While it's fine with cksum as it wouldn't output globs, imo wouldn't hurt to always quote right-hand for an external command output.

Reply:
... wrt other comment, plus if drop cksum it'll actually matter then.

Reply:
Guess could `diff/cmp <(... | sort) <(... | sort)` and check return value if diff is sure to be available, albeit I was slightly wary of this back in esed myself. cmp would be safe either way